input/examples/relatedperson-example-sex-and-gender.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
One of the examples needs two instances of gender identity. The RelatedPerson may be a good choice and use the period & comment in both instances. @reubendaniels may be able to supply meaningful input into the examples if the Gender Harmony IG does not offer sufficient background.

Reply:
Preliminary change done, and running it by @reubendaniels